examples/imgui-winit/Cargo.toml

```diff
@@ -16,5 +16,5 @@ imgui-wgpu = "0.12"
 imgui-winit-support = "0.6"
 log = "0.4"
 pixels = { path = "../.." }
-winit = "0.24"
-winit_input_helper = "0.9"
+winit = "0.23"
+winit_input_helper = "0.8"
```

Comment:
Hm, did the later versions of these introduce some sort of regression or unwanted behavior?

Reply:
Yes, the example wouldn't build at all, because winit_input_helper 0.9 requires winit 0.24, while other crates require winit 0.23. So I had to downgrade both of them to avoid a version conflict.

Reply:
@JoshLambda I just merged a patch that addresses the dependency issues in the `imgui-winit` example. Please rebase or merge on the latest HEAD to pick up the patch. It will conflict with this change, so you will want to revert.

Reply:
@parasyte Done.